lib/sidekiq/statistic/web_extension.rb

```diff
@@ -34,12 +34,15 @@ def self.registered(app)
 
         app.get '/statistic/charts.json' do
           charts = Sidekiq::Statistic::Charts.new(*calculate_date_range(params))
+          date = {
```

Comment:
Is this a kind of information that would be better placed inside `Sidekiq::Statistic::Charts` itself?